src/formatting/lists.rs

```diff
@@ -537,10 +544,21 @@ where
     for item in items.clone().into_iter().skip(i) {
         let item = item.as_ref();
         let inner_item_width = UnicodeSegmentation::graphemes(item.inner_as_ref(), true).count();
+        let post_comment_is_multiline = item
+            .post_comment
+            .as_ref()
+            .map_or(false, |s| s.trim().contains('\n'));
```

Comment:
kind of a bummer to have to do the multi vs. single line check multiple times, but at first glance I don't see a good way of avoiding that

Reply:
Tried to find solution, but one of the checks whether the comment is mutliline is done in a loop for identifying the group, and the other is done later when formatting each items.  Adding `post_comment_is_multilne` field to `ListItem` and initialize it in `Iterator for ListItem` may work, but I didn't try to do such change.